client/src/tpu_client.rs

```diff
@@ -1,6 +1,104 @@
+use {
+    crate::connection_cache::ConnectionCache,
+    solana_connection_cache::connection_cache::ConnectionCache as BackendConnectionCache,
+    solana_rpc_client::rpc_client::RpcClient,
+    solana_sdk::{
+        message::Message,
+        signers::Signers,
+        transaction::{Transaction, TransactionError},
+        transport::Result as TransportResult,
+    },
+    solana_tpu_client::tpu_client::temporary_pub::Result,
+    std::sync::Arc,
+};
 pub use {
     crate::nonblocking::tpu_client::TpuSenderError,
     solana_tpu_client::tpu_client::{
-        TpuClient, TpuClientConfig, DEFAULT_FANOUT_SLOTS, MAX_FANOUT_SLOTS,
+        TpuClient as BackendTpuClient, TpuClientConfig, DEFAULT_FANOUT_SLOTS, MAX_FANOUT_SLOTS,
```

Comment:
Move `TpuClient as BackendTpuClient` to L11 (the non-pub use statement)

Reply:
done